### ISSUE TITLE
What's wrong with Zammad 6 on docker compose ?

### ISSUE THREAD

**Issue by kamilkobak** [zammad/zammad-docker-compose#365]
I am trying to run Zammad 6 on docker compose and still having problems. As if variables are not passed between containers,  for example Postgres variables. I tested on Debin 12 and Ubuntu Server 22.04. 
If I run Zammad 5.4 on the same systems there is no problem.

# Infos
```shell
root@test2:~/zammad-docker-compose# docker version
Client: Docker Engine - Community
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.6
 Git commit:        ced0996
 Built:             Fri Jul 21 20:35:18 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.5
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.6
  Git commit:       a61e2b4
  Built:            Fri Jul 21 20:35:18 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.22
  GitCommit:        8165feabfdfe38c65b599c4993d227328c231fca
 runc:
  Version:          1.1.8
  GitCommit:        v1.1.8-0-g82f18fe
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

```shell
root@test2:~/zammad-docker-compose# cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

```shell
root@test2:~/docker/zammad-docker-compose# cat .env 
ELASTICSEARCH_VERSION=8.8.0
IMAGE_REPO=ghcr.io/zammad/zammad
MEMCACHE_SERVERS=zammad-memcached:11211
MEMCACHE_VERSION=1.6.20-alpine
POSTGRES_DB=zammad_production
POSTGRES_PASS=zammad
POSTGRES_USER=zammad
POSTGRES_HOST=zammad-postgresql
POSTGRES_PORT=5432
POSTGRES_VERSION=15.3-alpine
REDIS_URL=redis://zammad-redis:6379
REDIS_VERSION=7.0.5-alpine
RESTART=always
VERSION=6.0.0-66
```

# Steps to reproduce the behavior
```shell
git clone https://github.com/zammad/zammad-docker-compose.git
cd zammad-docker-compose
docker compose pull
docker compose up -d && docker compose logs -f
```

## Logs:
```
zammad-docker-compose-zammad-postgresql-1     | 2023-08-11 06:30:42.293 UTC [68] FATAL:  password authentication failed for user "zammad"
zammad-docker-compose-zammad-postgresql-1     | 2023-08-11 06:30:42.293 UTC [68] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"

```

## Checking variables:
```shell
root@test2:~/docker/zammad-docker-compose# docker compose exec zammad-railsserver cat config/database.yml
production:
  adapter: nulldb
  database: zammad_production
  pool: 50
  timeout: 5000
  encoding: utf8
  username: postgres
  password: 
  host: zammad-postgresql
```

Why ?

**Comment by pgassmann** [comment]
The current docker-compose.yml is broken. Postgresql client config is missing.
See: https://github.com/zammad/zammad-docker-compose/pull/357

Also upgrade to version 6 requires manual migration of the data. see https://github.com/zammad/zammad-docker-compose/issues/348#issuecomment-1609202582

**Comment by MrGeneration** [comment]
> There are many problems with zammad 6. I'm surprised this version has been released.


Please don't confuse compose / docker Zammad with Zammad itself. Zammad 6 itself is running great.

**Comment by kamilkobak** [comment]
OK thx

**Comment by mgruner** [comment]
Should be fixed by #366

**Comment by mgruner** [comment]
Solved by [7f31a9a](https://github.com/zammad/zammad-docker-compose/commit/7f31a9acfc19c970bdf1b1014946d862fe19f5fe).
